redirects.yml

```diff
@@ -14,8 +14,8 @@ redirects:
   'solutions/observability/apps/application-performance-monitoring-apm.md': 'solutions/observability/apm/index.md'
   'solutions/observability/apps/get-started-with-apm.md': 'solutions/observability/apm/get-started.md'
   'solutions/observability/apps/get-started-apm-serverless.md': 'solutions/observability/apm/get-started.md'
-  'solutions/observability/apps/fleet-managed-apm-server.md': 'solutions/observability/apm/apm-server-fleet-managed.md'
```

Comment:
Are we sure we want to remove the existing redirect? Are they already in use?

Should `solutions/observability/apm/apm-server-fleet-managed.md` also be redirected to `solutions/observability/apm/apm-server/fleet-managed.md` ?

Reply:
I was operating under the assumption that I should go back and edit existing redirects that are pointing to a filepath that is now going to redirect to prevent a chain of redirects. Pleeeease let me know if my assumption is incorrect!